docs/development/doc_guide.rst

```diff
@@ -1,76 +1,829 @@
+checks at the bottom of a pull request, and clicking on ``Details`` next
+to ``docs/readthedocs.org:plasmapy``.
 
 Building documentation
 ======================
-Documentation is built from the main branch on every commit pushed
-to it.
 
-Sphinx, the documentation generator of PlasmaPy, uses reStructuredText (reST) as its markup language. A primer on reST is available at this `webpage
-<https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html>`_
-of Sphinx's website.
+There are two methods for building the documentation: make_ and tox_.
+
+* Using make_ will build the documentation based off of what is in the
+  current directory structure. Using make_ is quicker for local builds
+  but requires that setting up all dependencies ahead of time.
+* Using tox_ does not require setting up all dependencies ahead of time,
+  but is more computationally intensive since it builds the package and
+  environment before building the documentation. Consequently, PlasmaPy
+  uses tox_ for building the documentation on continuous integration
+  testing platforms.
+
+Prerequisites
+-------------
+
+To install all dependencies required to develop PlasmaPy on your local
+computer, enter the top-level directory of the cloned repository and
+run:
+
+.. code-block:: bash
+
+   pip install -r requirements.txt
+
+Building documentation with make
+--------------------------------
 
-Using sphinx within the project
+If you have make_ installed, then you can build the documentation by
+entering the :file:`docs/` directory and running:
+
+.. code-block:: bash
+
+   make html
+
+Including the ``-j n`` flag in the ``make`` command will enable a
+parallel build, where ``n`` is replaced with the number of processes.
+
+To remove all files from a documentation build, run:
+
+.. code-block:: bash
+
+   make clean
+
+This command is needed when you make a change to a file that does not
+trigger Sphinx_ to rebuild the file that you altered.
+
+Building documentation with tox
 -------------------------------
-To build docs locally, either:
 
-* use `Tox <https://tox.readthedocs.io/en/latest/>`_ with ``tox -e build_docs`` from within the main PlasmaPy repository directory, or
-* enter the ``docs`` directory and run ``make html``.
+You can use tox_ to locally build the documentation by running:
 
-Afterwards, open ``docs/_build/html/index.html`` with your browser of choice.
+.. code-block:: bash
 
-Do try to solve warnings in documentation when writing your code. To enforce this,
-The ``build_docs`` environment is set to fail on encountering any warnings via
-the ``-W`` flag to ``sphinx-build``
+   tox -e build_docs
 
-.. note::
-   The ``tox -e build_docs_no_examples`` command will build the documentation without
-   executing the :ref:`example notebooks <example_notebooks>`. It will also
-   pass with warnings.
+You can access the documentation landing page by opening
+:file:`docs/_build/html/index.html` with your browser of choice.
```

Comment:
You should make a similar comment under the previous section, "Building documentation with make."